runtime/setup.py

```diff
@@ -424,6 +424,8 @@ def generate_version_py():
             "iree-run-trace*",
             "iree-benchmark-module*",
             "iree-benchmark-trace*",
+            "iree-dump-module*",
+            "iree-cpuinfo*",
```

Comment:
@stellaraccident was `iree-cpuinfo` intentionally excluded from this
```
    # Matching the native extension as a data file keeps setuptools from
    # "building" it (i.e. turning it into a static binary).
    package_data={
```
section in the previous PR? I can leave that and/or `iree-dump-module` out of this list if so.

Reply:
Not intentionally